src/Nethermind/Nethermind.Core/Transaction.cs

```diff
@@ -111,13 +111,12 @@ private void ClearPreHashInternal()
         public int DataLength => Data?.Length ?? 0;
 
         public AccessList? AccessList { get; set; } // eip2930
+
         public UInt256? MaxFeePerDataGas { get; set; } // eip4844
+
         public byte[]?[]? BlobVersionedHashes { get; set; } // eip4844
 
-        // Network form of blob transaction fields
-        public byte[]? BlobKzgs { get; set; }
-        public byte[]? Blobs { get; set; }
-        public byte[]? BlobProofs { get; set; }
+        public object? NetworkWrapper { get; set; }
```

Comment:
Does it make sense to be un-typed?

Reply:
This certain additional abstraction is semantically cleaner imho and costed almost nothing, because we still need to check for null which is usually the same as using `is`. It's just some wrapper until we need to validate a blob tx.